src/components/portrait/portrait.spec.js

```diff
@@ -0,0 +1,382 @@
+import React from 'react';
+import ReactTestUtils from 'react-dom/test-utils';
+import { shallow } from 'enzyme';
+import { ThemeProvider } from 'styled-components';
+
```

Comment:
empty line

Reply:
Yes, this is intentional. I like to add an empty line to separate [the NPM packages](https://github.com/Sage/carbon/blob/dd2fab1/src/components/portrait/portrait.spec.js#L1-L4) from [the local modules](https://github.com/Sage/carbon/blob/dd2fab1/src/components/portrait/portrait.spec.js#L6-L13). But I'm happy to remove it if you wish.

Reply:
We didn't do that before so I think will be good to keep everything like 'line under line' without spaces

Reply:
OK, updated in fc49c74 :thumbsup: